variables.tf

```
@@ -9,6 +9,11 @@ variable "cluster_kubernetes_version" {
   default     = "v1.21.4+rke2r2"
 }
 
+variable "additional_manifest" {
```

It might make sense to name the variable `cluster_additional_manifest` since the module generally uses that prefix for variables which relate only to the RKE2 cluster config.

```suggestion
variable "cluster_additional_manifest" {
```